mon-pix/app/templates/components/new-information.hbs

```diff
@@ -0,0 +1,14 @@
+<div class="new-information">
+      <div class="new-information__content">
+        <img class="new-information-content__img"
+             src="{{this.rootURL}}/images/Illu_niveau6_{{t 'current-lang'}}.svg"
+             alt=""
+             role="presentation"/>
+        <div class="new-information-content__text">
+          {{{@information}}}
+        </div>
+      </div>
+  <button class="new-information__close" {{action @closeaction}} type="button">
+    {{fa-icon 'times' size='lg'}}<span class="sr-only">{{t 'common.actions.close'}}</span>
```

Comment:
**nitpick (non-blocking):** `@size='lg'` doit pouvoir marcher sinon je propose d'utiliser un `font-size=1.33rem` (cf https://fontawesome.com/how-to-use/on-the-web/styling/sizing-icons)
```suggestion
    <FaIcon @icon='times' @size='lg'/><span class="sr-only">{{t 'common.actions.close'}}</span>
```

Reply:
Modifié !